web/components/TextBlock/SimpleCallToAction/SimpleCallToAction.tsx

```diff
@@ -12,6 +12,7 @@ type SimpleCallToActionProps = {
 
 export const SimpleCallToAction = ({
   value: { text, reference },
-}: PortableTextComponentProps<SimpleCallToActionProps>) => (
-  <ButtonLink text={text} url={`/${reference?.slug?.current}`} />
-);
+}: PortableTextComponentProps<SimpleCallToActionProps>) => {
+  const url = reference?.slug?.current ? `/${reference?.slug?.current}` : '#';
+  return <ButtonLink text={text} url={url} />;
```

Comment:
Nit: Should we instead return `null` (render nothing) if the url is not set?

Reply:
Good point! 126c84c